src/index.html

```diff
@@ -9,6 +9,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <img class="image"
+        src="images/logo.png"
+        alt="Moyo">
```

Comment:
```suggestion
        <img 
          class="image"
          src="images/logo.png"
          alt="Moyo"
        >
```

Consider fixing indentations accordingly to the code style guide